tests/unit/channel_tests.py

```diff
@@ -392,6 +392,26 @@ def test_basic_qos_raises_channel_closed(self):
         self.assertRaises(exceptions.ChannelClosed, self.obj.basic_qos, 0,
                           False, True)
 
+    def test_basic_qos_raises_prefetch_size_raises_error(self):
+        self.obj._set_state(self.obj.OPEN)
+        with self.assertRaises(ValueError) as ex:
+            self.obj.basic_qos('foo', 123)
+        self.assertEqual("invalid literal for int() with base 10: 'foo'",
+                         ex.exception.args[0])
+        with self.assertRaises(ValueError) as ex:
+            self.obj.basic_qos(-1, 123)
+        self.assertIn('prefetch_size', ex.exception.args[0])
+
+    def test_basic_qos_raises_prefetch_count_raises_error(self):
```

Comment:
Something doesn't look right with the method name - the word "raises" before "_prefetch_count". Did you mean to name it `test_basic_qos_*invalid*_prefetch_count_raises_error`?